appveyor.yml

```diff
@@ -11,6 +11,7 @@ init:
       Import-Module '..\appveyor-tool.ps1'
 
 install:
+  - cmd: set PATH=C:\Ruby27-x64\bin;%PATH%

```

Comment:
@Yuri05 Trying to set a new version of Ruby explicitely and reverting your changes from before